src/main/java/Main.java

```diff
@@ -0,0 +1,9 @@
+import static spark.Spark.*;
+
+public class Main {
+
+  public static void main(String[] args) {
+    get("/hello/:name", (request, response) -> "Hello " + request.params(":name"));
```

Comment:
Create a package called users inside java package, and then a package called controllers inside users. 
In package controllers create a file called `usersController.java` and move the `get` method in this main to that file.
Then Instantiate the class `usersController.java` in this main method.

By now, you can test it and should be working the same way.